cmd/integrationArtifactDeploy.go

```diff
@@ -99,7 +99,11 @@ func runIntegrationArtifactDeploy(config *integrationArtifactDeployOptions, tele
 		log.Entry().
 			WithField("IntegrationFlowID", config.IntegrationFlowID).
 			Info("successfully deployed into CPI runtime")
-		deploymentError := pollIFlowDeploymentStatus(retryCount, config, httpClient, serviceKey.OAuth.Host)
+		taskId, readErr := ioutil.ReadAll(deployResp.Body)
+		if readErr != nil {
+			return errors.Wrap(readErr, "HTTP response body could not be read")
+		}
+		deploymentError := pollIFlowDeploymentStatus(string(taskId), retryCount, config, httpClient, serviceKey.OAuth.Host)
```

Comment:
is taskId not already a string?

Reply:
its byte array as output of  ioutil.ReadAll